index.html

```diff
@@ -1328,8 +1328,8 @@
 allows for in-person intrusions, because it can reveal detailed information
 about a person's life) but it might also be public and not sensitive at all, or
 it might be low-enough granularity that it is less sensitive for many
-people. Reducing granularity of location information may not hide precise location information from
-an [=actor=], however, particularly if location information is repeatedly requested over time or if
+people. Beware that reducing granularity of location information may not hide precise location information from
```

Comment:
Oh, and avoiding normative language:

```suggestion
people. Beware that reducing granularity of location information might not hide precise location information from
```